src/www/ui/api/Models/Analysis.php

```diff
@@ -392,8 +478,11 @@ public function getArray($version = ApiVersion::V1)
         "monk"      => $this->monk,
         "nomos"     => $this->nomos,
         "ojo"       => $this->ojo,
+        "scanoss"   => $this->scanoss,
         "reso"      => $this->reso,
         "package"   => $this->pkgagent,
+        "patent"    => $this->ipra,
+        "heritage" => $this->softwareHeritage,
```

Comment:
```suggestion
        "ipra"    => $this->ipra,
        "softwareHeritage" => $this->softwareHeritage,
```

Please keep naming consistent.

Reply:
done